src/components/About.astro

```diff
@@ -98,16 +99,17 @@ const images = [
       <div>
         <div class="swiper w-full xl:w-[500px] 2xl:w-[700px]">
           <div class="swiper-wrapper">
-            {images.map((image) => (
-              <div class="swiper-slide !h-72 !w-72 xl:!h-[525px] xl:!w-[525px]">
+            {images.map(({caption, src}) => (
+              <div class="swiper-slide !h-72 !w-72 xl:!h-[555px] xl:!w-[525px]">
```

Comment:
What are the implications of changing this height? Did you test out some sample captions and did this look good?

Reply:
Yes, I did exactly that. They don't show at all unless it's taller. Any thoughts on potential issues?

Reply:
Should be fine for now 👍 . Mainly just worried about captions of different sizes on different screen sizes etc